.ci/smoke-tests.groovy

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env groovy
+@Library('apm@current') _
+
+pipeline {
+  agent { label 'linux && immutable' }
+  environment {
+    REPO = 'apm-aws-lambda'
+    BASE_DIR = "src/github.com/elastic/${env.REPO}"
+    AWS_ACCOUNT_SECRET = 'secret/observability-team/ci/elastic-observability-aws-account-auth'
+    EC_KEY_SECRET = 'secret/observability-team/ci/elastic-cloud/observability-pro'
+    TERRAFORM_VERSION = '1.2.3'
+    CREATED_DATE = "${new Date().getTime()}"
+    SLACK_CHANNEL = "#apm-serverless"
+  }
+
```

Comment:
To bypass an existing issue while using params for the very first build, this should avoid an empty/unset env variable in those cases

```suggestion
    SMOKETEST_VERSIONS = "${params.SMOKETEST_VERSIONS}"
  }
```